code/workspaces/infrastructure-api/src/kubernetes/__snapshots__/ingressGenerator.spec.js.snap

```diff
@@ -116,3 +141,28 @@ spec:
           servicePort: 8000
 "
 `;
+
+exports[`Ingress generator should not add auth-url for a non-private endpoints 1`] = `
+"---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: name-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/auth-url: http://localhost:9000/auth
```

Comment:
Should the `auth-url` be in this snapshot?

Reply:
Well spotted...fixed the test